sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/ingestion/models/one/CommonSchemaLogSerializerTest.java

```diff
@@ -108,6 +108,12 @@ public void serializeAndDeserialize() throws JSONException {
         log.getExt().getLoc().setTz("-08:00");
         checkSerialization(serializer, log);
 
+        /* Device extension. */
```

Comment:
We opted to keep same JSON order as bond as One Collector team some Asimov customers are sensitive to JSON order of fields.

Device extension is located between user and os extensions in Bond.